templates/pwm.asg.element.json

```diff
@@ -0,0 +1,881 @@
+                                        ]
+                                    ]
+                                }
+                            },
+                            "61-chmod-conf": {
+                                "command": "chmod 640 /opt/aws/amazon-cloudwatch-agent/etc/amazon-cloudwatch-agent.json"
+                            },
+                            "62-chown-conf": {
+                                "command": "chown root:root /opt/aws/amazon-cloudwatch-agent/etc/amazon-cloudwatch-agent.json"
+                            },
+                            "70-inject-custom-agent-config": {
+                                "command": {
+                                    "Fn::Join": [
+                                        "",
+                                        [
+                                            "/opt/aws/amazon-cloudwatch-agent/bin/",
+                                            "amazon-cloudwatch-agent-ctl",
+                                            " -a fetch-config -m ec2 -c",
+                                            " file:/opt/aws/amazon-cloudwatch-agent/etc/",
+                                            "amazon-cloudwatch-agent.json -s"
+                                        ]
+                                    ]
+                                }
+                            }
+                        }
+                    },
+                    "c-bootstrap": {
+                        "commands": {
+                            "10-install-epel": {
+                                "command": "yum -y install epel-release"
+                            },
+                            "11-enable-epel": {
+                                "command": "yum-config-manager --enable epel"
+                            },
+                            "20-watchmaker-prep": {
+                                "command": "yum -y --enablerepo=epel install wget python-pip && bash -xe /etc/cfn/scripts/watchmaker-install.sh"
+                            },
+                            "30-watchmaker-launch": {
+                                "command": "watchmaker --log-level debug -n --log-dir=/var/log/watchmaker"
+                            },
+                            "40-clonepwmformula": {
+                                "command": {
+                                    "Fn::Join": [
+                                        "",
+                                        [
+                                            "git clone https://github.com/",
+                                            {
+                                                "Ref": "GitHubUsername"
+                                            },
+                                            "/",
+                                            {
+                                                "Ref": "GitHubRepo"
+                                            },
+                                            " /tmp/",
```

Comment:
This is because I don't include the GitHubRepo folder. I have it setup so that it will go straight to `/srv/salt/pillar` instead of `/srv/salt/<GitHubRepo>/pillar`

Reply:
this should be relevant to the salt state location, `file_roots`, not pillar. it is convention with salt formulas to place each formula directory into the `file_roots` config.